modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -913,10 +913,15 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	name = "Basil Uniform"
 	item_path = /obj/item/clothing/under/costume/basil_boys
 
+//ht
```

Comment:
This doesn't really provide any further insight, it's kinda just there to be there and those comments are pretty useless.
```suggestion
```

Reply:
eh fair